www/src/components/layout.js

```diff
@@ -58,11 +59,8 @@ class DefaultLayout extends React.Component {
   }
 
   render() {
-    const {
-      location = {
-        pathname: `/starter-showcase`,
-      },
-    } = this.props // location will be undefined if on 'starter-showcase'
+    let { location } = this.props
```

Comment:
Doing a `console.log(this.props.location.pathname)` here should show the double slash when running `gatsby build`.